Automattic/woocommerce-payments
# Add customer details flow to inital state

Fixes #7417 

#### Changes proposed in this Pull Request

This PR ads a customer detail flow to the initial state where we create a customer or update the customer's data before we proceed with the payment processing







#### Testing instructions







* Make sure that the customer is created if it's not existed
* Make sure that the customer is updated if it exists
* Make sure that customer is added to the order before payment is processed

-------------------


**Post merge**
